### PR TITLE
Created ProgressBar. Fixed unwanted rendering of the svg bar.

### PR DESCRIPTION
**Pull request by RamSaravana**
Created Progress-bar which is reusable. Made a logic which changes color of the bar based on the value provided. Fixed the unwanted rendering of the progress bar.

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -3,9 +3,11 @@ import AreaChart from "./components/AreaChart";
 import { useEffect, useState } from "react";
 import * as d3 from "d3";
 import Plate from "./components/Plate";
+import ProgressBar from "./components/ProgressBar";
 
 const Home = () => {
-  const [data, setData] = useState([]);
+  const [AreaChartData, setAreaChartData] = useState([]);
+  const [ProgressChartData, setProgressChartData] = useState([]);
```
Comment (binal-sasi) [inline src/Home.js:10]: We usually use `CapitalizedNaming` for classes and components. variables are usually `lowerCaseNamed`. This is not a breaking condition, as long as the team agrees. Here `areaChartData` and `progressChartData` would probably be better.

File: src/components/ProgressBar.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useRef } from "react";
+import * as d3 from "d3";
+import "./ProgressBar.css";
+
+const CheckColor = (value) => {
+  if (value < 75) return "red";
+  if (value > 90) return "green";
+  return "darkViolet";
+};
```
Comment (binal-sasi) [inline src/components/ProgressBar.js:9]: functions should be camel-cased. `checkColor` instead of `CheckColor`

**Review by binal-sasi — APPROVED** [review]
There are comments concerning naming conventions. Other than that, it looks good to me.

File: src/Home.js (hunk 1 above)
Comment (RamSaravana) [reply src/Home.js:10]: Noted it Binal. Will work based on the naming conventions as mentioned.

File: src/components/ProgressBar.js (hunk 2 above)
Comment (RamSaravana) [reply src/components/ProgressBar.js:9]: Got it. will make the changes from here on.

**Pull request merged**